check_sites.py

```diff
@@ -13,19 +13,35 @@
 # limitations under the License.
 from FpsCheck import FpsCheck
 import json
+import getopt
+import sys
 from publicsuffix2 import PublicSuffixList
 
 
 def main():
+    args = sys.argv[1:]
+    inputFile = 'first_party_sets.JSON'
+    inputPrefix = ''
+    opts, _ = getopt.getopt(args, "i:")
+    for opt, arg in opts:
+        if opt == '-i':
+            inputFile = arg
+            inputPrefix='main/'
```

Comment:
Can we supply this as another argument (e.g. `data_directory`), rather than hardcoding it?

Alternatively, you could derive it from the path of this "binary", using `sys.executable` and `os.path.split`.

Reply:
Applied the suggestion to supply this as argument "--data_directory"